docs/stdlib/abstract.rst

```diff
@@ -15,11 +15,11 @@ that can be applied to a broad range of types.
 
     :index: any anytype
 
-    Generic type.
+    Represents a generic type.
```

Comment:
In these instances where you've changed the description to read "_represents_" a particular type, is that accurate, or _are_ these that type? Use of the word "represents" makes me think these are some sort of stand-in for an actual type, but I believe they _are_ these types.

Reply:
"Represents" is added just to inform users that this is the actual representation of the type's definition and construct in the documentation

Reply:
I don't think this should define the documentation though. We're documenting the type, so the description should describe the type, not the documentation of the type.